website/docs/language/values/variables.mdx

```diff
@@ -128,7 +128,7 @@ collections:
 The keyword `any` may be used to indicate that any type is acceptable. For
 more information on the meaning and behavior of these different types, as well
 as detailed information about automatic conversion of complex types, see
-[Type Constraints](/language/expressions/types).
+[Type Constraints](/language/expressions/type-constraints).
 
 If both the `type` and `default` arguments are specified, the given default
 value must be convertible to the specified type.
```

Comment:
```suggestion
If you specify both the `type` and `default` arguments, the given default
value must be convertible to the specified type.
```

Suggest changing "are specified" to "If you specify" to avoid as passive voice style issue.